### ISSUE TITLE
Stop method missing

### ISSUE THREAD

**Issue by mariusrak** [garyhtou/toggl-track#10]
I cannot find a stop method that is described here
https://developers.track.toggl.com/docs/api/time_entries/index.html#patch-stop-timeentry

**Comment by garyhtou** [comment]
Hey @mariusrak, thanks for opening this issue! Looks like this endpoint may have been added after I added the TimeEntry resource/class.

I implemented the method in https://github.com/garyhtou/toggl-track/pull/11, but am having trouble successfully using the endpoint. Toggl seems to always respond with 500 "JSON is not valid" regardless of how I make the HTTP request. Would you mind testing out the endpoint? (such as by using [Insomia](https://insomnia.rest/) or [Postman](https://www.postman.com/))

**Comment by mariusrak** [comment]
Hi, I also have troubles with the endpoint. I tried to use some internals of your package and did this:
```js
return await toggl.request(`workspaces/${workspace_id}/time_entries/${currentId}/stop`, {
        method: "PATCH",
});
```
But, even though I got response with full time entry object, the timer on toggl didn't stop.

So I copied example from docs:
```js
const headers = {
        "Content-Type": "application/json",
        Authorization: `Basic ${base64encode(token + ":api_token")}`,
};
const url = `https://api.track.toggl.com/api/v9/workspaces/${workspace_id}/time_entries/${currentId}/stop`;
return await fetch(url, { method: "PATCH", headers }).then(resp => resp.json());
```

And this works.

Looking at the commit, I see that it misses `/stop` in the url, but still, there seems to be some sort of issue with underlaying code in your package.

**Comment by garyhtou** [comment]
Hi @mariusrak, I just released v0.8.0 which now supports the STOP method. Thanks for pointing out the missing `/stop` in the URL.

After updating to the lastest version (v0.8.0), you should be able to use the new method:

```TypeScript
import { Toggl } from 'toggl-track';

const toggl = new Toggl({
  auth: {
    token: 'YOUR TOKEN HERE',
  },
});

const currentEntry = await toggl.timeEntry.current();
console.log(currentEntry);

const stoppedEntry = await toggl.timeEntry.stop(currentEntry.id, 'YOUR WORKSPACE ID HERE');
console.log(stoppedEntry);
```

**Comment by mariusrak** [comment]
Yes, it's working. Thanks
